website/docusaurus.config.js

```diff
@@ -6,7 +6,7 @@ const editUrl = 'https://github.com/yewstack/yew/blob/master/website/'
 module.exports = {
     title: 'Yew',
     tagline:
-        'Yew is a modern Rust framework for creating multi-threaded front-end web apps with WebAssembly.',
+        'A framework for creating safe, reliable and efficient web applications.',
```

Comment:
I think it reads better without "safe". Otherwise, LGTM

Reply:
I concur. The word "safe" is ambiguous here anyway.

Reply:
Removed safe